<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scrambler</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="head_title_label">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>–®–∏—Ñ—Ä–∞—Ç–æ—Ä</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="catalog_label">
       <property name="text">
        <string>–ö–∞—Ç–∞–ª–æ–≥ –¥–ª—è —à–∏—Ñ—Ä–∞—Ü–∏–∏:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="catalog_line_edit"/>
       </item>
       <item>
        <widget class="QPushButton" name="catalog_choose_button">
         <property name="text">
          <string>üìÅ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="remember_button">
         <property name="text">
          <string>–ó–∞–ø–æ–º–Ω–∏—Ç—å</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="catalog_label_2">
       <property name="text">
        <string>–ö–∞—Ç–∞–ª–æ–≥ –¥–ª—è –¥–µ—à–∏—Ñ—Ä–∞—Ü–∏–∏:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="decrypt_catalog_line_edit"/>
       </item>
       <item>
        <widget class="QPushButton" name="catalog_choose_button_2">
         <property name="text">
          <string>üìÅ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="decrypt_button">
         <property name="text">
          <string>–î–µ—à–∏—Ñ—Ä–æ–≤–∞—Ç—å</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
